src/Jackett.Common/Definitions/1337x.yml

```diff
@@ -10,6 +10,7 @@ links:
   - https://1337x.to/
   - https://1337x.gd/
   - https://1337x.st/
+  - https://1337xxx.to/
```

Comment:
for https://1337xxx.to/ I get redirected to  https://www.1337xxx.to/
does this happen to you?
if it does then please update. otherwise you will generate 
```
Exception (1337x): Got redirected to another domain. Try changing the indexer URL to https://www.1337xxx.to/. Updated site link, please try again.: Parse error
```